Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.yml

```diff
@@ -250,8 +250,8 @@ script:
       name: include_child_org_units
       description: Whether to return devices from all child organizational units. If this is set to true, 'org_unit_path' must be provided.
       predefined:
-      - yes
-      - no
+      - true
```

Comment:
double check this is switched in the code. check what changed this, seems like bug-prone behavior

Reply:
this argument is passed to argToBoolean which excepts both true false or yes no so should be fine